compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -771,4 +771,8 @@ public static void castToJavaString(MethodVisitor mv, int fieldNameRegIndex, int
 
     private JvmCodeGenUtil() {
     }
+
+    public static String getRefTypeConstantName(BTypeReferenceType type) {
```

Comment:
should this be part of jvmRefTypeconstantGenclass

Reply:
We use this in `JvmTypeGen` as well. So, I kept this method in the commonplace.